nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -510,6 +512,14 @@ define([
             }
 
             state.fileType.completed = newFileTypeState;
+            updateEditingState();
+        }
+
+        /**
+         * This will toggle the cell to either the editingComplete or editingIncomplete state,
+         * based on the ready state of each filetype.
+         */
+        function updateEditingState() {
             let cellReady = true;
```

Comment:
Factored this out of updateParameterState so it can be called separately.

Also, in the upcoming PR where we move the file path chooser code to the configure tab, the updateParameterState function will go with it.